widget/ui/src/components/Tabs/Tabs.types.tsx

```diff
@@ -0,0 +1,24 @@
+import type { Tabs } from './Tabs.styles';
+import type * as Stitches from '@stitches/react';
+import type { ReactNode } from 'react';
+
+type BaseProps = Stitches.VariantProps<typeof Tabs>;
+type BaseType = Exclude<BaseProps['type'], object>;
+type BaseBorder = Exclude<BaseProps['border'], object>;
+type Item = {
+  id: string | number;
+  title?: string;
```

Comment:
I believe that in each situation, it is necessary for either the 'title' or 'icon' to be required. Now they both can be undefined. You can improve that by replacing it with union of two types.

Reply:
fixed